pandas/core/indexes/datetimes.py

```diff
@@ -2237,6 +2238,21 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
     Returns
     -------
     rng : DatetimeIndex
+
+    See Also
+    ________
```

Comment:
Consider using `----` for consistency

Reply:
Ohh silly mistake! thanks for figuring it out